lib/fog/aws/requests/data_pipeline/activate_pipeline.rb

```diff
@@ -14,16 +14,20 @@ def activate_pipeline(id)
 
           response = request({
             :body => Fog::JSON.encode(params),
-            :headers => { 'X-Amz-Target' => 'DataPipeline.ActivatePipeline' },
+            :headers => { 'X-Amz-Target' => 'DataPipeline.ActivatePipeline' }
           })
-
-          Fog::JSON.decode(response.body)
```

Comment:
We are changing the return value for all of these requests to the raw response from the decoded body.  While I think this is good direction, I would like @fog/fog-aws to comment on the relevance of the breaking change.